src/Symfony/Component/Ldap/Security/LdapUser.php

```diff
@@ -60,8 +60,16 @@ public function getUserIdentifier(): string
         return $this->identifier;
     }
 
+    /**
+     * @deprecated since Symfony 7.3
+     */
+    #[\Deprecated(since: 'symfony/security-core 7.3')]
     public function eraseCredentials(): void
     {
+        if (\PHP_VERSION_ID < 80400) {
+            @trigger_error(\sprintf('Method %s::eraseCredentials() is deprecated since symfony/security-core 7.3', self::class), \E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(\sprintf('Method %s::eraseCredentials() is deprecated since symfony/security-ldap 7.3', self::class), \E_USER_DEPRECATED);
```